packages/next/test/fixtures/00-app-dir/package.json

```
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "next": "https://files-26yo0dy1b-ijjk-testing.vercel.app",
+    "next": "12.2.6-canary.10",
```

```suggestion
    "next": "canary",
```
I think we can just use canary here to make sure we don't break this support across releases